start_admin_sshd.sh

```diff
@@ -98,6 +119,9 @@ for key_alg in rsa ecdsa ed25519; do
   # If both of the keys exist, don't overwrite them
   if [[ -s "${SSH_HOST_KEY_DIR}/ssh_host_${key_alg}_key" ]] \
   && [[ -s "${SSH_HOST_KEY_DIR}/ssh_host_${key_alg}_key.pub" ]]; then
+    if [[ ${use_eic} == 1 ]]; then
```

Comment:
Nit. It's safe to remove this conditional and just symlink regardless, but I'm also fine with it as is.